nengo_spa/algebras/vtb.py

```diff
@@ -58,10 +58,22 @@ def invert(cls, v):
         return v.reshape((sub_d, sub_d)).T.flatten()
 
     @classmethod
-    def get_binding_matrix(cls, v):
+    def get_binding_matrix(cls, v, swap_inputs=False):
```

Comment:
I'm not sure I understand the point of swapping inputs as a function argument. Naively, it seems to me that the inputs should just be swapped in the connections and not as a parameter, if the user wants that. Would you mind explaining to me the intent of this?

Reply:
When using a binding matrix one vector is fixed (the one the matrix) is based on and one varies over time. Thus, there is no way to swap the inputs because there is only on time varying input.